metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  "hectcastro@gmail.com"
 license           "Apache 2.0"
 description       "Installs and configures collectd."
-version           "2.0.0"
+version           "2.0.1"
```

Comment:
Can you please omit this from the pull request? I'll bump the version along with other changes included in the next release.